### PR TITLE
healer

### PR DESCRIPTION
**Pull request by minders14**

### CODE REVIEW

File: KTLOSNaxxLoot.json [hunk 1]
```diff
@@ -216,7 +216,7 @@
 			"wowID":"23030"
 		},
 		"Noth's Frigid Heart":{
-			"prio":["Peku","LC","EP/GP"],
+			"prio":["Peku2","Nisio(2)","LC","EP/GP"],
```
Comment (zhang165) [inline KTLOSNaxxLoot.json:219]: Can I has first? Using fire runed grimore...

**Review by zhang165 — CHANGES_REQUESTED** [review]

File: KTLOSNaxxLoot.json (hunk 1 above)
Comment (minders14) [reply KTLOSNaxxLoot.json:219]: Only if I get the priest ring before you :P

Comment (zhang165) [reply KTLOSNaxxLoot.json:219]: Ring of faith?
Sure... but I'm not even on it lol

Comment (minders14) [reply KTLOSNaxxLoot.json]: Done

**Review by zhang165 — APPROVED** [review]

**Pull request merged**